conda_smithy/lint_recipe.py

```diff
@@ -660,7 +660,9 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
         expected_exceptions = set([l.strip() for l in expected_exceptions])
     if set(filtered_licenses) - expected_licenses:
         hints.append(
-            "License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression."
+            "License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.\n\n"
+            "A full list of acceptable licenses can be found "
+            "[here]( https://spdx.org/licenses/ )."
```

Comment:
I think we should link to out documentation at https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

Reply:
Thanks! Changed :)